src/bot/index.ts

```diff
@@ -58,11 +58,11 @@ export async function bot({ currency, family, mutation }: Arg = {}) {
   const maybeCurrency = currency
     ? findCryptoCurrencyByKeyword(currency)
     : undefined;
-  const maybeFilterOnlyFamily = family;
+  const maybeFilterOnlyFamily = family ? family.split(",") : undefined;
 
-  for (const family in allSpecs) {
-    const familySpecs = allSpecs[family];
-    if (maybeFilterOnlyFamily && maybeFilterOnlyFamily !== family) {
+  for (const fam in allSpecs) {
+    const familySpecs = allSpecs[fam];
+    if (maybeFilterOnlyFamily && !maybeFilterOnlyFamily.includes(fam)) {
```

Comment:
From L61, it seems `maybeFilterOnlyFamily` can be `undefined` right?

Reply:
I think it shouldn't be a problem we got if (maybeFilterOnlyFamily) covering the case it's undefined or maybe i misunderstood the point ?